src/tidysic.py

```diff
@@ -19,19 +19,67 @@ def guess_file_metadata(filename):
         if separator > 0:
             artist = filename[0:separator].lstrip()
             title = filename[separator + 2:len(filename)].lstrip()
-
-            return (artist, title)
+            
+            if guess_file_metadata.accept_all:
+                return (artist, title)
+            else:
+                # ask user what to do
+                message = [
+                    f"Guessed {artist}, {title} ",
+                    f"Accept (y)",
+                    f"Accept all (a)",
+                    f"Discard (d)",
+                    f"Rename (r)"
+                    ]
+                log(message, prefix="Asking", color="orange1")
+                answer = input("answer : ")
```

Comment:
Writes two lines for a single question.

The following suggestion is not stylised as the rest of the output, but this can be arranged if we override the input method during the `log` refactor.
```suggestion
                answer = input("(y/a/d/r) ")
```